net-dialup/radiusclient/files/radiusclient-0.3.2-BJA-cross.diff

```diff
@@ -0,0 +1,67 @@
+--- configure.in	2002-02-03 14:51:40.000000000 +0100
```

Comment:
Where does this bug come from? It is a good practice to leave some notes at top of the patch, like what it does and where it comes from, if it's committed upstream, if it has upstream bug etc.